team.py

```diff
@@ -40,4 +42,20 @@ def edit_password(self, input):
             return False
         self.password = password
         ret = "Password successfully changed to " + password
-        return ret
\ No newline at end of file
+        return ret
+
+    def answer_question(self, user_answer):
+        if self.database.get_current_user() is not self:
+            return False
```

Comment:
One method returns two different types: booleans and strings. Might lead to some confusion and tricky syntax wherever this is returned